libraries/stdlib/samples/test/samples/collections/iterables.kt

```diff
@@ -24,7 +24,13 @@ class Iterables {
 
     @Sample
     fun iterable() {
-        val iterator = Iterable { buildIterator { yieldAll(listOf(1, 2, 3)) } }.iterator()
+        val iterator = buildIterator {
```

Comment:
Sorry, I don't see `Iterable` function call here again. Have you forgotten for which function this sample is? :)

What I've meant is something like:
```
val someIterable = Iterable { buildIterator { ... } }

// now you can call extensions on the Iterable
val result: List<String> = someIterable.mapIndexed { index, value -> "$index => $value" }
assertPrints(result, "...")
```

Reply:
Really sorry!! I will try to fix samples.

Reply:
Thanks, fixed.